pages/gallery.html

```diff
@@ -36,32 +36,32 @@
     </br>
     </br>
     <main>
-        <div class="slideViewer" id="changeSlideLoad">
-<span class="imagesG" onclick="setSlide(1)" >[1]</span>
-<span class="imagesG" onclick="setSlide(2)" >[2]</span>
-<span class="imagesG" onclick="setSlide(3)" >[3]</span>
-<span class="imagesG" onclick="setSlide(4)" >[4]</span>
-<span class="imagesG" onclick="setSlide(5)" >[5]</span>
-<span class="imagesG" onclick="setSlide(6)" >[6]</span>
-<span class="imagesG" onclick="setSlide(7)" >[7]</span>
-<span class="imagesG" onclick="setSlide(8)" >[8]</span>
-<span class="imagesG" onclick="setSlide(9)" >[9]</span>
-<span class="imagesG" onclick="setSlide(10)" >[10]</span>
-<span class="imagesG" onclick="setSlide(11)" >[11]</span>
-<span class="imagesG" onclick="setSlide(12)" >[12]</span>
-<span class="imagesG" onclick="setSlide(13)" >[13]</span>
-<span class="imagesG" onclick="setSlide(14)" >[14]</span>
-<span class="imagesG" onclick="setSlide(15)" >[15]</span>
-<span class="imagesG" onclick="setSlide(16)" >[16]</span>
-<span class="imagesG" onclick="setSlide(17)" >[17]</span>
-<span class="imagesG" onclick="setSlide(18)" >[18]</span>
-<span class="imagesG" onclick="setSlide(19)" >[19]</span>
-<span class="imagesG" onclick="setSlide(20)" >[20]</span>
-<span class="imagesG" onclick="setSlide(21)" >[21]</span>
-<span class="imagesG" onclick="setSlide(22)" >[22]</span>
+     <div class="slideViewer" > 
+      <span class="imagesG" onclick="setSlide(1)" >[1]</span>
```

Comment:
Please revert back formatting changes - we do not want to have formatting changes here. They should be made in a separate commit.